### PR TITLE
Setup mobile version Pull request

### PR DESCRIPTION
**Pull request by mar4444**
What I did:

- Setting of Linters
- Created html file which is index.html
- Created css file which is style.css
- GitHub flow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- ABOT HEADER -->
+    <header class="nav-bar">
+        <nav class="nav-section">
+            <p class="logo">My Logo</p>
```
Comment (OyePriscilla) [inline index.html:17]: - [ ] Kindly wrap the logo with `a` tag, this is for convention, semantics, and user experience(UX) reasons. 👍 Kindy bring the logo out of the `nav` container, a `logo` is not supposed to be part of `nav` items. 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            </div>
+        </nav>
+    </header>
+    <!-- ABOUT MYSELF -->
+    <main class="about">
+        <h2 class="about-title">I’m Martin.Glad to see you!</h2>
+        <p class="par-me">
+            I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
```
Comment (OyePriscilla) [inline index.html:31]: - [ ] Kindly reduce the width and font-weight of this pargraph in accordance to the given design, reproducing perfectly a given design shows how well we pay attention to details. 👍 

## Your Design
Untitled

## Figma Design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @mar4444 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used flexbox to placed elements in the page ✔️ 
Used images and backgrounds to enhance the look of the website ✔️ 

## Required Changes ♻️

- [ ] Kindly fix the `issues` broken link in the `contributing` section of the README file. 👍 I was redirected to a wrong page, perhaps you [meant](https://github.com/mar4444/re-portfolio/issues)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <div class="btn">
+                <div class="btn-1"></div>
+                <div class="btn-1"></div>
+                <div class="btn-1"></div>
+            </div>
+        </nav>
+    </header>
+    <!-- ABOUT MYSELF -->
+    <section class="about">
+        <h1 class="about-title">I’m Martin.Glad to see you!</h1>
```
Comment (Qoosim) [inline index.html:27]: - [OPTIONAL] I would be perfect if you could break this line in order to make it look perfectly as we have it on the Figma design across different screen sizes.
```
   <h1 class="about-title">I’m Martin. <br> Glad to see you!</h1>
```

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @mar4444, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**